src/app/Layout/Layout.tsx

```diff
@@ -0,0 +1,74 @@
+"use client"
+import React, {  useState } from 'react'
+import {
+    MagnifyingGlassIcon,
+    HamburgerMenuIcon,
+} from '@radix-ui/react-icons'
+import { Input } from "@/components/ui/input"
+import { Button } from "@/components/ui/button"
+import { Sidebar } from '../components/Sidebar'
+import { usePathname } from 'next/navigation'
+import { RoutesTitle } from '@/lib/routes'
```

Comment:
This variable name makes no sense. It gives impression that there's only one title.